Kitodo/src/main/java/org/kitodo/production/forms/CommentForm.java

```diff
@@ -323,4 +290,40 @@ public void newComment() {
         setCommentMessage("");
         setCorrectionComment(false);
     }
+
+    /**
+     * Check whether there are concurrent tasks in work or not.
+     *
+     * @return whether there are concurrent tasks in work or not
+     */
+    public boolean isConcurrentTaskInWork() {
+        return this.currentTask.getId() < 0 || !getConcurrentTasksInWork().isEmpty();
```

Comment:
Can id be smaller than 0?

Reply:
You are correct, this is some residue from an earlier version of the code, where I saved the current tasks ID instead of the whole task in the CommentForm. I removed the check.